documentation/writing_plans.md

```diff
@@ -896,11 +896,25 @@ without_default_logging { run_command('echo hi', $targets) }
 
 For information on configuring log levels, see [Logs](logs.md).
 
+### Log functions
+
+Bolt ships with built-in functions for logging at each of Bolt's log levels.
+
+| Log level | Plan function |
+| --- | --- |
+| `trace` | [`log::trace`](plan_functions.md#logtrace) |
+| `debug` | [`log::debug`](plan_functions.md#logdebug) |
+| `info` | [`log::info`](plan_functions.md#loginfo) |
+| `warn` | [`log::warn`](plan_functions.md#logwarn) |
+| `error` | [`log::error`](plan_functions.md#logtrace) |
+| `fatal` | [`log::fatal`](plan_functions.md#logfatal) |
+
 ### Puppet log functions in Bolt
 
 You can use Puppet log functions in Bolt plans, but Bolt log levels do not map
 directly to Puppet log levels. For example, a `notice` function in a plan logs
-at the `info` level in Bolt. Log levels map as follows:
+at the `info` level in Bolt. Whenever possible, use Bolt's log functions instead
+of Puppet's. Log levels map as follows:
```

Comment:
```suggestion
at the `info` level in Bolt. Whenever possible, use Bolt log functions instead
of Puppet log functions. Log levels map as follows:
```
Generally, we try to avoid using product names in the possessive. It's a style thing that I tend to be kind of lenient on, but in this case I think it's better to avoid it.